tests/unit/test_python_evaluation.py

```diff
@@ -12,6 +13,8 @@
     recall_at_k,
     ndcg_at_k,
     map_at_k,
+    _merge_ranking_true_pred,
```

Comment:
mmmm, I don't know about testing private methods. In SOF https://stackoverflow.com/a/105021 they say that if a private method is complex enough that needs to be tested, the user make it public or a wrapper around it. However, maybe we don't want to make that function public...

@msalvaris any suggestion here?